src/components/FactsContainer/index.js

```diff
@@ -4,34 +4,41 @@ import { useSelector } from "react-redux";
 import Fact from "./Fact";
 import usePagination from "../../hooks/usePagination";
 import Loading from "../Loading";
+import EmptyContainer from "../EmptyContainer";
 
 const FactsContainer = () => {
   const state = useSelector((state) => state);
-  const { allFacts, isLoading } = state;
-  const { pagination, onChange, pageSize } = usePagination(allFacts, 9);
+  const { filtredFacts, isLoading } = state;
```

Comment:
You can destructure this state in place.
Like
const {filteredFacts, isLoading} = useSelector((state) => state);